packages/generator/README.md

```diff
@@ -112,7 +112,27 @@ Autocomplete will also show the available properties of the data returned by SDK
 
 ## Caching
 
-The SDK currently supports two types of caches - In-memory and Redis. Both the implementations respect [standard cache headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control). If you would like to use another type of cache, you can write your own implementation of the [CacheManager](../core/src/base/cacheManager.ts)
+The SDK currently supports two types of caches - In-memory and Redis. Both the implementations respect [standard cache headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control). If you would like to use another type of cache, you can write your own implementation of the [CacheManager](../core/src/base/cacheManager.ts). You can refer to this [default cache manager](../core/src/base/cacheManagerKeyv.ts) when designing your implementation.  
+
+### Cache storage adapter
+
+The default cache storage limits to 10,000 distinct entities before applying a simple [least recently used](https://en.m.wikipedia.org/wiki/Cache_replacement_policies#Least_Recently_Used_.28LRU.29) policy for cache replacement. You can change the limit by creating a [quick-lru](https://www.npmjs.com/package/quick-lru) storage adapter.
+```javascript
+import { CacheManagerKeyv } from "@commerce-apps/core";
+import { QuickLRU } from "quick-lru";
+
+const cacheManagerKeyv = new CacheManagerKeyv({ keyvStore: new QuickLRU({ maxSize: 50000 })});
```

Comment:
We might want to call out more explicitly that this is optional to pass and not required.

Reply:
maxSize is required https://github.com/sindresorhus/quick-lru#readme

Reply:
Looks like their readme is wrong. Their code requires options.maxSize > 0

Reply:
I was talking about from our perspective it can me make clearer what happens when you don't specify a cache manager at all.   Again not blocking definitely can close.